submissions/maryOkosun/php/loop.php

```diff
@@ -0,0 +1,13 @@
+<?php                               //A program to return every number between 100 ad 150
+    for($i = 100; $i  <=150; $i++ ){
+        echo "$i <br>";
+    }
+?>
+
+<?php                              //A program to return every even number between 0 and 50
```

Comment:
I think you should remove 
```
?>

<?php

``` 
on line 5 and 7

Reply:
Thanks @adaugochi 
I have removed the unnecessary tags and resolved the issues with the first program. Please review